dev/sg/sg_setup.go

```diff
@@ -1085,53 +1083,53 @@ func getChoice(choices map[int]string) (int, error) {
 }
 
 func retryCheck(check dependencyCheck, retries int, sleep time.Duration) dependencyCheck {
-	return func(ctx context.Context) (ok bool, err error) {
+	return func(ctx context.Context) (err error) {
 		for i := 0; i < retries; i++ {
-			ok, err = check(ctx)
-			if ok {
-				return true, nil
+			err = check(ctx)
+			if err != nil {
+				return nil
```

Comment:
That's a bug, I think:

```suggestion
				return err
```

Reply:
Oh thank you for spotting my mistake :pray:.